docs/content/api-overview/resources/load-balancer.md

```diff
@@ -0,0 +1,120 @@
+| loadBalancingRule | backend_address_pool | The name of the backend address pool to which this rule applies. |
+| loadBalancingRule | probe | The name of the probe to use to check the health of services in the backend pool. |
+| loadBalancingRule | frontend_port | The port on the frontend to forward to a backend service. |
+| loadBalancingRule | backend_port | The target port on the backend service. |
+| loadBalancingRule | protocol | The protocol to forward, defaults to 'All'. |
+| loadBalancingRule | idle_timeout_minutes | The time in minutes before a TCP connection is considered idle and disconnected. |
+| loadBalancingRule | load_distribution_policy | The load distribution policy - 'Default' where a request can go to any backend, 'SourceIP' which is mapped on client IP, or 'SourceIPProtocol' which is mapped on client IP and protocol. |
+| loadBalancingRule | enable_tcp_reset | After an idle timeout, the TCP connection is reset - defaults to 'disabled'. |
+| loadBalancingRule | enable_outbound_snat | Allows backend services to use this load balancer for outbound connections - defaults to 'disabled'. |
+
+#### Example
+
+```fsharp
+open Farmer
+open Farmer.Builders
+open Farmer.LoadBalancer
+
+arm {
+    add_resources [
+        vnet {
+            name "my-vnet"
+            add_address_spaces [ "10.0.1.0/24" ]
+            add_subnets [
+                subnet {
+                    name "my-services"
+                    prefix "10.0.1.0/24"
+                    add_delegations [
+                        SubnetDelegationService.ContainerGroups
+                    ]
+                }
+            ]
+        }
+        loadBalancer {
+            name "lb"
+            sku Sku.Standard
+            add_frontends [
+                frontend {
+                    name "lb-frontend"
+                    public_ip "lb-pip"
+                }
+            ]
+            add_backend_pools [
+                backendAddressPool {
+                    name "lb-backend"
+                    link_to_vnet "my-vnet"
+                    add_ip_addresses [
+                        "10.0.1.4"
+                        "10.0.1.5"
+                    ]
+                }
+            ]
+            add_probes [
+                loadBalancerProbe {
+                    name "httpGet"
+                    protocol LoadBalancerProbeProtocol.HTTP
+                    port 8080
+                    request_path "/"
+                }
+            ]
+            add_rules [
+                loadBalancingRule {
+                    name "rule1"
+                    frontend_ip_config "lb-frontend"
+                    backend_address_pool "lb-backend"
+                    frontend_port 80
+                    backend_port 8080
+                    protocol TransmissionProtocol.TCP
+                    probe "httpGet"
+                }
+            ]
+            add_dependencies [
+                Farmer.Arm.Network.virtualNetworks.resourceId "my-vnet"
```

Comment:
This shouldn't be required I think - should just be able to take in the VNet as an IBuilder and work from that?